2016-05-03 15:09:08.9131 | Error | AppDomain未处理异常。 System.InvalidOperationException: 没有发现将要向外公开的服务接口,请确保您用的是Thrift生成的服务接口
   在 Cicada.Rpc.Server.RpcServer.RunPrivate[T]() 位置 e:\liu\workspace\net\Cicada\branches\1.0\src\Cicada.Rpc\Server\RpcServer.cs:行号 54
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart() 
 ModuleBoot.OnUnhandledException => ModuleBoot.DoException => Log.Error 

2016-05-03 15:09:14.1258 | Error | AppDomain未处理异常。 System.InvalidOperationException: 没有发现将要向外公开的服务接口,请确保您用的是Thrift生成的服务接口
   在 Cicada.Rpc.Server.RpcServer.RunPrivate[T]() 位置 e:\liu\workspace\net\Cicada\branches\1.0\src\Cicada.Rpc\Server\RpcServer.cs:行号 54
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart() 
 ModuleBoot.OnUnhandledException => ModuleBoot.DoException => Log.Error 

2016-05-03 15:09:21.4662 | Error | AppDomain未处理异常。 System.InvalidOperationException: 没有发现将要向外公开的服务接口,请确保您用的是Thrift生成的服务接口
   在 Cicada.Rpc.Server.RpcServer.RunPrivate[T]() 位置 e:\liu\workspace\net\Cicada\branches\1.0\src\Cicada.Rpc\Server\RpcServer.cs:行号 54
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart() 
 ModuleBoot.OnUnhandledException => ModuleBoot.DoException => Log.Error 

2016-05-03 15:09:38.7993 | Error | AppDomain未处理异常。 System.InvalidOperationException: 没有发现将要向外公开的服务接口,请确保您用的是Thrift生成的服务接口
   在 Cicada.Rpc.Server.RpcServer.RunPrivate[T]() 位置 e:\liu\workspace\net\Cicada\branches\1.0\src\Cicada.Rpc\Server\RpcServer.cs:行号 54
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart() 
 ModuleBoot.OnUnhandledException => ModuleBoot.DoException => Log.Error 

2016-05-03 15:10:38.9138 | Error | AppDomain未处理异常。 System.InvalidOperationException: 没有发现将要向外公开的服务接口,请确保您用的是Thrift生成的服务接口
   在 Cicada.Rpc.Server.RpcServer.RunPrivate[T]() 位置 e:\liu\workspace\net\Cicada\branches\1.0\src\Cicada.Rpc\Server\RpcServer.cs:行号 54
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart() 
 ModuleBoot.OnUnhandledException => ModuleBoot.DoException => Log.Error 

2016-05-03 15:13:34.8985 | Error | AppDomain未处理异常。 System.InvalidOperationException: 没有发现将要向外公开的服务接口,请确保您用的是Thrift生成的服务接口
   在 Cicada.Rpc.Server.RpcServer.RunPrivate[T]() 位置 e:\liu\workspace\net\Cicada\branches\1.0\src\Cicada.Rpc\Server\RpcServer.cs:行号 54
   在 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   在 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   在 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   在 System.Threading.ThreadHelper.ThreadStart() 
 ModuleBoot.OnUnhandledException => ModuleBoot.DoException => Log.Error 

2016-05-03 15:17:44.6678 | Error | Thrift.Transport.TTransportException: System.Net.Sockets.SocketException (0x80004005): 一个封锁操作被对 WSACancelBlockingCall 的调用中断。
   在 System.Net.Sockets.Socket.Accept()
   在 System.Net.Sockets.TcpListener.AcceptTcpClient()
   在 Thrift.Transport.TServerSocket.AcceptImpl()
   在 Thrift.Transport.TServerSocket.AcceptImpl()
   在 Thrift.Transport.TServerTransport.Accept()
   在 Thrift.Server.TThreadedServer.Serve() 
